### PR TITLE
Prompt logged out user to login

### PR DESCRIPTION
**Pull request by wooleyHeadspring**
When the API determines that the user's authentication has expired and returns a 401 the component should send the user to the login page instead of just displaying an error.

### CODE REVIEW

File: src/Web/src/utils/logout-service.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import config from '../config';
+
+function LogoutService() {
+    const history = useHistory();
+    const { logoutAuth, getAuthInfo } = AuthService();
+    const authInfo = getAuthInfo();
+    const { API_URL } = config();
+
+    function logout() {
+        const apiUrl = new URL(API_URL.href + '/account/logout');
```
Comment (YogirajA) [inline src/Web/src/utils/logout-service.js:12]: Do you want to start using the URL constructor correctly here or fix all of them later? I am OK either way.

Comment (wooleyHeadspring) [reply src/Web/src/utils/logout-service.js:12]: I think capturing it and fixing them all at once later is the way to go.

**Review by YogirajA — CHANGES_REQUESTED** [review]
@wooleyHeadspring can you resolve conflicts?

**Comment by wooleyHeadspring** [comment]
> @wooleyHeadspring can you resolve conflicts?

Resolved by merging main into this branch.

**Review by YogirajA — APPROVED** [review]

**Pull request merged**